pyiron_atomistics/gpaw/pyiron_ase.py

```diff
@@ -26,7 +25,7 @@
 __date__ = "Sep 1, 2018"
 
 
-class AseJob(GenericInteractive):
+class AseJob(GenericInteractive, ABC):
```

Comment:
Why is the `AseJob` now an abstract class? I thought the use case of this class is that ASE atoms which have a corresponding calculator object attached can be executed directly inside pyiron using this class.

Reply:
It is abstract because whenever we run it we need to call `set_calculator` and set calculator raises a not implemented error. Ie you can't just instantiate this class and use it, you _must_ subclass and override a particular method. That is exactly the point of an abstract class with an abstract method, so I am just making it official so it's clearer to future developers.